packages/components/src/components/primitives/TextPairing/TextPairing.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { TextPairing, TextPairingVariant, textPairingVariants } from '.';
+import { Text } from 'components';
+import { Box, Flex, Spacer, Stack, StackDivider } from '@chakra-ui/layout';
+
+export default {
+  component: TextPairing,
+  title: 'Components/Primitives/TextPairing',
```

Comment:
What do you think about moving this component to Molecules instead of Primitives?